### PR TITLE
Parametric  dmd

### PR DESCRIPTION
**Pull request by kevinhkhuynh**

### CODE REVIEW

**Comment by kevinhkhuynh** [comment]
RFR, but can wait until PR #95 is finished reviewing and merged.

**Comment by kevinhkhuynh** [comment]
This is ready for review.

File: lib/linalg/Vector.h
Comment (dylan-copeland) [inline lib/linalg/Vector.h:798]: Pass by reference?

File: lib/linalg/Vector.h
Comment (dylan-copeland) [inline lib/linalg/Vector.h:791]: Pass by reference?

File: lib/algo/ParametricDMD.h
Comment (dylan-copeland) [inline lib/algo/ParametricDMD.h:67]: Pass by reference?

File: lib/algo/ParametricDMD.C
Comment (dylan-copeland) [inline lib/algo/ParametricDMD.C:49]: Long line (needs to be broken).

File: examples/dmd/parametric_heat_conduction.cpp
Comment (dylan-copeland) [inline examples/dmd/parametric_heat_conduction.cpp:324]: This line and several others in this file are too long and should be broken.

**Review by dylan-copeland — APPROVED** [review]

**Review by chldkdtn — APPROVED** [review]

**Comment by chldkdtn** [comment]
@siuwuncheung Would you do review for this PR?

**Review by siuwuncheung — APPROVED** [review]

**Comment by siuwuncheung** [comment]
> @siuwuncheung Would you do review for this PR?

Just approved. 👍

**Pull request merged**
